lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -1176,25 +1174,19 @@ class VerticalLineLabel extends FlLineLabel with EquatableMixin {
   /// Drawing text will retrieve through [labelResolver],
   /// you can override it with your custom data.
   /// [show] determines showing label or not.
+  /// [direction] determines if the direction of the text should be horizontal or vertical.
   VerticalLineLabel({
-    EdgeInsets? padding,
-    TextStyle? style,
-    Alignment? alignment,
-    bool? show,
-    String Function(VerticalLine)? labelResolver,
-  })  : labelResolver =
-            labelResolver ?? VerticalLineLabel.defaultLineLabelResolver,
-        super(
-          show: show ?? false,
-          padding: padding ?? const EdgeInsets.all(6),
-          style: style ??
-              const TextStyle(
-                color: Colors.black,
-                fontWeight: FontWeight.bold,
-                fontSize: 14,
-              ),
-          alignment: alignment ?? Alignment.bottomRight,
-        );
+    super.padding = const EdgeInsets.all(6),
+    super.style = const TextStyle(
+      color: Colors.black,
+      fontWeight: FontWeight.bold,
+      fontSize: 14,
+    ),
+    super.alignment = Alignment.bottomRight,
+    super.show = false,
+    super.direction = LabelDirection.horizontal,
```

Comment:
Does it make sense to have a vertical direction by default here?

Reply:
I realized that the default behavior is horizontal. So we don't need to change it. 
Thanks for keeping it the same as before.